### ISSUE TITLE
Issues with addplot when adding scatter plot

### ISSUE THREAD

**Issue by Chadonillo** [matplotlib/mplfinance#77]
Hi, I am very new to this Library. I enjoy the simplicity of it, but I am having some issues.
I simply want to draw arrows on my candlestick graph just like you did: https://github.com/matplotlib/mplfinance/blob/master/examples/addplot.ipynb

If I try to add a scatter plot it shrinks the Y-axis shrinks massively. I added pictures so you can see.
![ScreenClip  2](https://user-images.githubusercontent.com/42037280/78390672-da10a400-75dc-11ea-9544-30c4133d0f65.png)
![ScreenClip](https://user-images.githubusercontent.com/42037280/78390616-c06f5c80-75dc-11ea-9eff-5d7d2801fb48.png)

```
line = mpf.make_addplot(myData["High"],scatter=False) #No issues drawing a line
scatter = mpf.make_addplot(myData["High"],scatter=True) #Issues drawing scatter
    
mpf.plot(myData,
               type = 'candle',
               style='yahoo', 
               addplot= line,
               figscale=1.8)
```

**Comment by DanielGoldfarb** [comment]
Hi.  Try adding `secondary_y = False` to your make_addplot() call:

    scatter = mpf.make_addplot(myData["High"],scatter=True,secondary_y=False)

If that doesn't work, then please attach your code, and a csv file with your data, and I will try to reproduce and see what's wrong.

All the best.   --Daniel

**Comment by DanielGoldfarb** [comment]
@Chadonillo ... sorry for the delay; other projects and things taking time.  hoping to look at this next week sometime.

**Comment by Chadonillo** [comment]
Okay, no problem! Thank you for the update. Hope to hear from you soon again.

**Comment by DanielGoldfarb** [comment]
I'm looking into this now.  It appears to have something to do with the way the underlying `matplotlib.axes.Axes.scatter()` scales the y-axis differently from `matplotlib.axes.Axes.plot()`.  

I have not yet figured out why this issue does not affect any of the plots **[in my addplot] examples(https://github.com/matplotlib/mplfinance/blob/master/examples/addplot.ipynb)**, but does affect these plots.

I will continue looking to see what I can figure out, and whether there is a workaround.   I initially tried this as a workaround: https://stackoverflow.com/a/19916667/1639359 ... but that didn't fix it, and even caused some issues with other `mplfinance` addplots that use `scatter=True`.   I will keep looking into this.  **So far** it appears that nothing is wrong with your code, and nothing is wrong with your data.

**Comment by DanielGoldfarb** [comment]
I have just added a quick-fix for this issue.  It is not on Pypi (and will not be for another week or so) but you can grab the change by cloning this repository and installing it locally.

There are now two additional kwargs to `mpf.plotting()`.  They are `set_ylim` (for the main panel) and `set_ylim_panelB` (for the lower panel).  Both kwargs expect a list or tuple of length two, where the entries are int or float, in the order `ymin, ymax`.  Thus for example:

    scatter = mpf.make_addplot(myData["High"],scatter=True,secondary_y=False)
    mpf.plot(myData,type = 'candle',style='yahoo',addplot= scatter,set_ylim=(1.1271,1.1286))

See also, for example, **[this notebook](https://github.com/matplotlib/mplfinance/blob/master/examples/scratch_pad/issues/Issue77_test.ipynb)**

This will, for the time being, allow you to override whatever `mplfinance` decides to do for the y-axis limits, while we come up with a better more permanent solution.   Let me know if you have any questions.   All the best.  --Daniel

**Comment by Chadonillo** [comment]
Thank you for your great support and time. That works perfectly for what I need!

**Comment by DanielGoldfarb** [comment]
I'm going to keep this open (and labeled bug).  I really want to figure out why this one case behaves differently, and it appears to have something to do with `Axes.plot()` handling `ylimits` differently than `Axes.scatter()` especially when both are on the same Axes object.

**Comment by free-soellingeraj** [comment]
I looked into this with the notebook provided.  Using Pandas 1.0.x latest, matplotlib latest, and mplfinance from master, I was unable to reproduce this problem.  

I was unable to reproduce the problem by reverting to Pandas 0.25.3 which the respondent used. 
 
However, by reverting matplotlib to version 3.0.0, I was able to reproduce the issue.  Here is the list of matplotlib versions I tested:
`matplotlib==3.0.0`: Bug existed
`matplotlib==3.1.0`: Bug existed
`matplotlib==3.2.0`: Bug existed
`matplotlib==3.2.1`: Bug Fixed

Based on this, I recommend the following: 
- Upgrading your `matplotlib` dependency to version 3.2.1 found here: https://github.com/matplotlib/matplotlib/releases/tag/v3.2.1 using 
`pip install matplotlib==3.2.1`
- Maybe we want to change the dependencies in `mplfinance/setup.py` to reflect the version restriction on `matplotlib>=3.2.1`, but I defer to the expert @DanielGoldfarb

**Comment by DanielGoldfarb** [comment]
@free-soellingeraj Aaron,

**Thanks!  Good job!**  

I had previously done some research into this here: https://github.com/matplotlib/mplfinance/blob/master/examples/scratch_pad/issues/Issue77_test.ipynb where I was able to reproduce the problem outside of `mpfinance` (in an attempt to show that it was specifically related to `matplotlib`) however I got distracted by other issues and had not gotten to the point of testing matplotlib versions.  In that notebook, I am using matplotlib v3.1.3 and pandas v1.0.3 and _did_ see the problem.)

I will try to make some time to see if I can reproduce your results (i.e. fix occurs at matplotlib v3.2.1).  Interestingly, I do not see anything here https://github.com/matplotlib/matplotlib/releases/tag/v3.2.1 that seems to me to suggest being related to the issue we see here.  But if I can reproduce finding the issue fixed with matplotlib v3.2.1 then perhaps @tacaswell can provide some insights as to why.
